sashimi-webapp/static/styles/markdown-pages.css

```diff
@@ -0,0 +1,7 @@
+@import url("/vendors/gutenberg-css/gutenberg.custom.css");
+@import url("/vendors/gutenberg-css/themes/oldstyle.min.css");
+@import url("/vendors/highlight.js/styles/ascetic.css");
+
+body {
+  text-align: initial;
+}
```

Comment:
Fixed, added newline at the end of the file.
Btw.. I found this an answer and question about adding newline at the end of file. 
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline